docker/production.yml

```diff
@@ -52,6 +52,7 @@ services:
       - WEB_URL
       - SEARCH_URL
       - FEED_DISCOVERY_URL
+      - STATUS_URL
```

Comment:
I thought @manekenpix said to remove this?

Reply:
I confused that with another declaration above, my mistake. Gonna clear this rn